code/datums/config/config_num.dm

```diff
@@ -0,0 +1,454 @@
+/decl/config/num
+	abstract_type = /decl/config/num
+	config_flags = CONFIG_FLAG_NUM | CONFIG_FLAG_HAS_VALUE
+	default_value = 0
+	var/min_value = -(INFINITY)
```

Comment:
From having experimented on my side, it seems most config that want a number will want an unsigned number. With a few exceptions. Some also just accept -1 as "disabled" value.

Reply:
That's fair. Just remarked it was a more common occurrence than accepting negatives.